client/src/templates/Challenges/classic/DesktopLayout.js

```diff
@@ -39,7 +39,7 @@ class DesktopLayout extends Component {
           <ReflexSplitter propagate={true} {...resizeProps} />
           <ReflexElement flex={1} {...resizeProps}>
             {challengeFile && (
-              <ReflexContainer key={challengeFile.key} orientation='horizontal'>
+              <ReflexContainer className='desktop-layout' key={challengeFile.key} orientation='horizontal'>
```

Comment:
@scissorsneedfoodtoo Let's do `desktop-layout` `id` on the outer `ReflexContainer`. What do you think?

Reply:
Yes, that's a much better idea! Thank you. Making the change now

Reply:
@ValeraS, I may have spoken too soon. While it seems like the `ReflexSplitter` component can take an `id`, `ReflexContainer` and `ReflexElement` cannot.

What do you think is best? We could stick with the class names for both `ReflexContaners` or we could wrap everything within the `Fragment` tags in a `div` and set its id or class name to `desktop-layout`. Or if you have another idea I'd be happy to test and implement that here

Reply:
Let's use `#learn-app-wrapper .desktop-layout` selector.

Reply:
Alright, sounds good. I'll go ahead and do that now